### ISSUE TITLE
Frontend: Improve transfer activity card design

### ISSUE THREAD

**Issue by GabrielBuragev** [beamer-bridge/beamer#1205]
<img width="199" alt="image" src="https://user-images.githubusercontent.com/18466333/203086994-d8bff1ba-ee6c-4aae-8edc-feff5e36e652.png">

# Rationale
As decided in the [UX Audit](ub.com/beamer-bridge/beamer/issues/1198) we want to improve the design of the transfer activity cards with the following:

- Remove third step ("Waiting for transaction receipt") from transfer status component.
- Introduce a new transfer status message "Waiting for user input" (or something similar) which signals that the user needs to confirm an action inside his wallet provider. This will make the differentiation between the passive/active steps clearer.
- Adjust step messages to use verbs. (Instead of "Waiting for transaction confirmation", use "Sign allowance transaction" or something similar). @DBecane your input would be really appreciated on this one.
- Hide the transfer summary (detail box containing the exact amount sent & transaction hash) until the request has been finalized (for both completed & failed).

**Comment by DBecane** [comment]
@GabrielBuragev Thanks for looping me in here! :)


As far as I remember, there are only 2 truly active steps required from users, right? In this case, the actions from users would happen at step: 

1. Token allowance
2. Signing transaction

So I would completely remove the "waiting..." since no action is required here. + I would rephrase the last one "Transfer in progress" So that the user knows that they should not close the window or interrupt the process in any way.

**Comment by GabrielBuragev** [comment]
@DBecane Thank you for your input. 
Let's use "Transfer in progress" instead of "Request is being fulfilled" then.

P.S. A transfer, on the front-end, cannot be interrupt by closing the browser session.
